README.md

```diff
@@ -73,36 +73,64 @@ Then, for more content, check out the other tutorials avaliable [here](https://g
 * And many more fixes, additions, and new material!
 
 ## Installation
-Snorkel uses Python 2.7 or Python 3 and requires [a few python packages](python-package-requirement.txt) which can be installed using [`conda`](https://www.continuum.io/downloads) and `pip`.
 
-### Setting Up Conda
-Installation is easiest if you download and install [`conda`](https://www.continuum.io/downloads).
-You can create a new conda environment with e.g.:
-```
-conda create -n py2Env python=2.7 anaconda
-```
-And then run the correct environment:
-```
-source activate py2Env
-```
+Starting with version 0.7.0, Snorkel should be installed as a Python package using pip.
+Snorkel can be installed directly from its GitHub repository via:
 
-### Installing dependencies
-First install [NUMBA](https://numba.pydata.org/), a package for high-performance numeric computing in Python via Conda:
-```bash
-conda install numba
 ```
-
-Then install the remaining package requirements:
-```bash
-pip install --requirement python-package-requirement.txt
+# WARNING: read sections below before running this command!
+# Note this installs the latest master version. You can change master to tag or commit
+pip install git+https://github.com/HazyResearch/snorkel@master
 ```
 
-Finally, enable `ipywidgets`:
-```bash
-jupyter nbextension enable --py widgetsnbextension --sys-prefix
+However, installing Snorkel via pip will not install dependencies.
+Instead, to manage its environment, Snorkel uses [conda](http://conda.pydata.org/docs/), which allows specifying dependencies via an `environment.yml` file.
+This documentation covers two common cases (usage and development) for setting up conda environments for Snorkel.
+In both cases, the environment can be activated using `conda activate snorkel` and deactivated using `conda deactivate`.
+With older versions of conda, you may have to replace `conda` with `source` in the above commands.
+
+### Usage Environment
```

Comment:
First instructions user should hit is how to install Snorkel as a user (dev environment can come lower in instructions).